packages/gatsby-seo/src/index.js

```diff
@@ -0,0 +1,114 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Helmet from "react-helmet"
+import { StaticQuery, graphql } from "gatsby"
+
+function SEO({
+  children,
+  description,
+  lang,
+  link,
+  meta,
+  render = children,
+  keywords,
+  title,
+  ...rest
+}) {
+  return (
+    <StaticQuery
+      query={detailsQuery}
+      render={data => {
+        const metaDescription =
+          description || data.site.siteMetadata.description
+        return (
+          <Helmet
+            htmlAttributes={{
+              lang,
+            }}
+            title={title}
+            titleTemplate={`%s | ${data.site.siteMetadata.title}`}
+            link={[].concat(link)}
+            meta={[
+              {
+                name: `description`,
+                content: metaDescription,
+              },
+              {
+                property: `og:title`,
+                content: title,
+              },
+              {
+                property: `og:description`,
+                content: metaDescription,
+              },
+              {
+                property: `og:type`,
+                content: `website`,
+              },
+              {
+                name: `twitter:card`,
+                content: `summary`,
+              },
+              {
+                name: `twitter:creator`,
+                content: data.site.siteMetadata.author,
```

Comment:
Hey 👋 I'm not sure if you actually want feedback on this already, so please excuse me if this is too early for me to comment or not my place at all (I'm a new maintainer).

I love the idea of a `<SEO>` component because it seems like a lot of the starters use something like this already! However, I'm not sure about the defaults for the Twitter meta tags. [According to the docs](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup.html), `twitter:creator` is only used in combination with `twitter:card` of type `summary`, so the way you have it right now isn't correct, according to their docs. Also, the username on Twitter is usually different than what people would expect to put in `siteMetadata.author`, so maybe it should be a separate field?

Reply:
Happy for feedback, whenever! If you have anything else--feel free to comment on whatever!

> twitter:creator is only used in combination with twitter:card of type summary

Isn't it used that way, currently? The username is/could be wrong (e.g. author name instead of twitter handle)--I was going to tweak that, but didn't want to introduce a breaking change (e.g. you have to augment your gatsby-config.js to take advantage of this component) for anyone picking this up.

Reply:
Oh my bad! I've been switching between projects where I had a similar but slightly issue for too long today, sorry. I understand your concern with the breaking change.